app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryCommentsFragment.kt

```diff
@@ -262,11 +265,20 @@ fun GalleryCommentsScreen(galleryDetail: GalleryDetail, navigator: NavController
         },
     ) { paddingValues ->
         val keylineMargin = dimensionResource(id = R.dimen.keyline_margin)
+        var editTextMeasured by remember { mutableStateOf(0.dp) }
         Box(modifier = Modifier.fillMaxSize().imePadding()) {
+            val additionalPadding = if (commenting) {
+                editTextMeasured
+            } else {
+                16.dp + 56.dp // Fab size + Fab space
+            }
```

Comment:
A bit too much when combined with the load more button

Reply:
Done